client/repoutils.c

```diff
@@ -182,6 +182,11 @@ TDNFRepoGetRpmCacheDir(
     if(access(pszRpmCacheDir, F_OK))
     {
         dwError = errno;
+        if (dwError == ENOENT)
+        {
+            dwError = 0;
+            pszRpmCacheDir = NULL;
```

Comment:
This pointer is allocated, call free here.

Reply:
done